content/en/blog/_posts/XXXX-XX-XX-fine-grained-supplementalgroups-control-beta/index.md

```diff
@@ -0,0 +1,195 @@
+
+# Verify that the Pod's Container is running:
+$ kubectl get pod strict-supplementalgroups-policy
+
+# Check the process identity:
+kubectl exec -it strict-supplementalgroups-policy -- id
+```
+
+The output should be similar to this:
+
+```none
+uid=1000 gid=3000 groups=3000,4000
+```
+
+You can see `Strict` policy can exclude group `50000` from `groups`! 
+
+Thus, ensuring `supplementalGroupsPolicy: Strict` (enforced by some policy mechanism) helps prevent the implicit supplementary groups in a Pod.
+
+{{<note>}}
+Actually, this is not enough because container with sufficient privileges / capability can change its process identity. Please see the following section for details.
+{{</note>}}
+
+## Attached process identity in Pod status
+
+This feature also exposes the process identity attached to the first container process of the container
+via `.status.containerStatuses[].user.linux` field. It would be helpful to see if implicit group IDs are attached.
+
+```yaml
+...
+status:
+  containerStatuses:
+  - name: ctr
+    user:
+      linux:
+        gid: 3000
+        supplementalGroups:
+        - 3000
+        - 4000
+        uid: 1000
+...
+```
+
+{{<note>}}
+Please note that the values in `status.containerStatuses[].user.linux` field is _the firstly attached_
+process identity to the first container process in the container. If the container has sufficient privilege
+to call system calls related to process identity (e.g. [`setuid(2)`](https://man7.org/linux/man-pages/man2/setuid.2.html), [`setgid(2)`](https://man7.org/linux/man-pages/man2/setgid.2.html) or [`setgroups(2)`](https://man7.org/linux/man-pages/man2/setgroups.2.html), etc.), the container process can change its identity. Thus, the _actual_ process identity will be dynamic.
+{{</note>}}
+
+## `Strict` Policy requires newer CRI versions
+
+Actually, CRI runtime(e.g. containerd, CRI-O) plays a core role for calculating supplementary group ids to be attached to the containers. Thus, `SupplementalGroupsPolicy=Strict` requires the CRI runtime that support this feature (`SupplementalGroupsPolicy: Merge` can work with the CRI runtime which does not support this feature because this policy is fully backward compatible policy).
+
+Below list is the popular CRI runtimes which supports this feature:
+
+- containerd: v2.0 or later
+- CRI-O: v1.31 or later
+
+And, you can see if the feature is supported in the Node's `.status.features.supplementalGroupsPolicy` field.
+
+```yaml
+apiVersion: v1
+kind: Node
+...
+status:
+  features:
+    supplementalGroupsPolicy: true
+```
+
+## The Behavioral Changes Introduced In Beta
+
+At alpha release, when a pod with `SupplementalGroupsPolicy=Strict` are scheduled to a node that does NOT support this feature(i.e. `.status.features.supplementalGroupsPolicy=false`), the pod's supplemental groups policy gets fallback-ed to the `Merge` policy _silently_.
```

Comment:
s/gets fallback-ed/falls back`

Reply:
fixed in [1a307da](https://github.com/kubernetes/website/pull/49944/commits/1a307da81b31428d8da1d6071bcccf77ad84c175)